tests/integration/test_service_manager.py

```diff
@@ -43,6 +46,24 @@ def test___grpc_service_without_discovery_service___start_service___service_host
     _validate_if_service_running_by_making_rpc(port_number)
 
 
+@pytest.mark.expect_discovery_service_stub_error(True)
```

Comment:
If we keep this marker, we should use its presence or absence as a flag instead of specifying a bool parameter.